src/server.rs

```diff
@@ -241,13 +244,41 @@ struct Cli {
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let contents = fs::read_to_string("./config.toml")
-        .expect("Something went wrong reading the file");
-
-    let config: Config = toml::from_str(&contents)
-        .expect("Could not deserialize the config file");
-
-    println!("{:#?}", config);
+    // Attempt to read the contents of the config file
+    let contents = fs::read_to_string("../../config.toml");
```

Comment:
I think we can try to have a `GetDefaultDataDir` with a canonical path for UNIX (debian / ubuntu) / MacOS by reading the env $HOME and concatenating with the platform-specific path for application config files.

See https://github.com/bitcoin/bitcoin/blob/master/src/common/args.cpp#L686 as example, what do you think?

Reply:
@ariard My latest commit addresses this. It will either look for the existing log path/file or create a new log path/file. I put this in `util.rs`.

Then server.rs calls this function when it runs the config parser. It prints the outputs to the log path which will be printed in stdout. 

It was a fun issue for me. Let me know what else you can see or any improvements. If not we can land, and ill open an issue about logging in general.